scripts/metrics/metrics_xml_to_csv.py

```diff
@@ -18,20 +18,26 @@
 
 CONFIG_FILE = "config/config.yml"
 DEFAULT_CTEST_LOG_FILENAME = 'Test.xml'
-global DEFAULT_CSV_FILE
+
+
+def now_wrap():
+    """
+    Wrapper function for now() for unit testing purposes.
+    @return: datetime.now()
+    """
+    return datetime.datetime.now()
 
 
 def _get_default_csv_filename():
     # Format default file name based off of date
-    global DEFAULT_CSV_FILE
-    now = datetime.datetime.now()
+    now = now_wrap()
     month = now.month
     if month < 10:
         month = f"0{now.month}"
     day = now.day
     if day < 10:
         day = f"0{now.day}"
-    DEFAULT_CSV_FILE = f"{now.year}_{month}_{day}.csv"
+    return f"{now.year}_{month}_{day}.csv"
```

Comment:
Since you are updating this file, this block can be simplified to:
```
return f"{now.year}_{now.month:02d}_{now.day:02d}"
```

Reply:
Oh neat, never knew that thanks